lib/lambda_ethereum_consensus/p2p/metadata.ex

```diff
@@ -0,0 +1,58 @@
+defmodule LambdaEthereumConsensus.P2P.Metadata do
+  @moduledoc """
```

Comment:
who sets the data that is stored in this genserver?

Reply:
I don't really know how can i handle that part actually forgot about it. what should i do about here

Reply:
I don't know. Maybe you should take a look at other clients to see where they take this data from.